c2corg_ui/static/js/map/map.js

```diff
@@ -126,58 +165,69 @@ app.MapController = function($scope, mapFeatureCollection) {
     this.addTrackImporter_();
   }
 
+  // advanced search mode
+  if (this.advancedSearch_) {
+    if (this.location_.hasParam('bbox')) {
+      var bbox = this.location_.getParam('bbox');
+      var extent = bbox.split(',');
+      if (extent.length == 4) {
+        // FIXME what if extent is not valid?
```

Comment:
Sure, it means simply ignore the invalid extent.
But what I meant was: how can we figure out that the provided extent is not valid?
I could for instance check that xmin is actually < than xmax etc. but that's a bit laborious. Or that the final extent items are numbers. Perhaps there's something in OL3 that does that kind of check (I have not found)?

Or maybe we can assume it's not that a big deal: if the extent is invalid, the search will obviously fail and that's it!?